cfpb/cfgov-refresh
# Remove and add a couple categories to the Implementation Resource group

Content wanted different categories under the Implementation Resources group

## Additions

- `Compliance aid` and `Official guidance`

## Removals

- `CFPB Bulletins and Statements` and `Implementation and Compliance Material`

## Testing

- Run `./cfgov/manage.py migrate`
- go to the Categories for a page and try to select one of the new ones.
- verify the old ones don't exist

## Review

- @richaagarwal 
- @kave 
- @sarahsimpson09 (Could you verify these are correct in the Additions/Removals section?)

## Notes

- there are no pages with the old categories and content isn't entering in any with them so a data migration is not needed.
